WifiART/src/structs/ZState.java

```diff
@@ -17,18 +17,58 @@
 package structs;
 
 public class ZState {
+
     // Parameters
     private int floor;
     private double heightAboveFloor; // Height is in meters.
     private double heightAboveFloorUncertainty;
-    /* The locationMovement field specifies whether the STA is expected to change location,
-       not expected to change location, or if this is unknown. */
+
+    /**
+     * The locationMovement field specifies whether the STA is expected to change location,
+     *  not expected to change location, or if this is unknown.
+     */
     private String locationMovement;
 
-    // TODO(dmevans) Add getter methods.
+
+    /**
+     * Get the floor number.
+     *
+     * @return the floor number.
+     */
+    public int getFloor()  {
+        return floor;
+    }
+
+    /**
+     * Get the height above the floor.
+     *
+     * @return the height above the floor (in meters)
+     */
+    public double getHeightAboveFloor() {
```

Comment:
Add unit to method name.

Reply:
Sure. Should I also include units for latitude and longitude? It doesn't look like the decoder does so.